src/mygrad/operation_base.py

```diff
@@ -119,8 +119,10 @@ def backward(self, grad, *, graph, _reduction=None, **kwargs):
             if not var.constant:
                 if not var._ops:
                     raise InvalidBackprop(
-                        "Part of the computational graph containing "
-                        "this tensor was 'cleared' prior to backprop."
+                        f"Part of the computational graph containing "
+                        f"this tensor, {var}, was 'cleared' prior to backprop.\n"
+                        f"It is recommended that you clear all computational graphs "
+                        f"and restart your computation."
```

Comment:
```suggestion
                        "Part of the computational graph containing "
                        f"this tensor, {var}, was 'cleared' prior to backprop.\n"
                        "It is recommended that you clear all computational graphs "
                        "and restart your computation."
```

Reply:
I like making these all f-strings, it is also how PyCharm auto-formats this